Dockerfile

```diff
@@ -36,11 +36,14 @@ RUN add-apt-repository -y ppa:cz.nic-labs/bird && \
         python-pip \
         python-pyasn1 \
         python-netaddr \
+        python-nose \
```

Comment:
I thought you'd just wanted to remove the whole RUN apt-get command mess from calico-build, and were willing to take the hit of it being slightly weird to install that stuff on calico/node.